daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Deployment.hs

```diff
@@ -95,20 +99,13 @@ fetchTest tools@Tools{..} = do
           ]
         callProcessSilent damlc ["validate-dar", fetchedDar]
 
--- | Using `daml inspect-dar`, discover the main package-identifier of a dar.
-getMainPidByInspecingDar :: Tools -> FilePath -> String -> IO String
-getMainPidByInspecingDar Tools{damlc} dar projName = do
-  stdout <- callProcessForStdout damlc ["inspect-dar", dar]
-  [grepped] <- pure $
-        [ line
-        | line <- lines stdout
-        -- expect a single line containing double quotes and the projName
-        , "\"" `isInfixOf` line
-        , projName `isInfixOf` line
-        ]
-  -- and the main pid is found between the 1st and 2nd double-quotes
-  [_,pid,_] <- pure $ splitOn "\"" grepped
-  return pid
+-- | Discover the main package-identifier of a dar.
+getMainPidOfDar :: FilePath -> IO String
```

Comment:
May I suggest to rename this to `getMainPkgIdOfDar`? `pid` reminds me of process identifiers. :bowtie: